### ISSUE TITLE
Tidy Data: Dates as Data Lesson Edits

### ISSUE THREAD

**Issue by diannamorganti** [LibraryCarpentry/lc-spreadsheets#125]
Two small improvements to the tidy data, dates as data section

Under the heading <strong> Preferred date format </strong>: The first sentence uses presumptions about whether the audience comprehended fully the previous area - in a vein similar to saying "clearly" or "plainly", it intends to say "As you can see..." (though it has a typo). 

Also, the assignment of "odd" lacks clarity - 

I would reword to "Storing dates in MONTH, DAY, and YEAR in separate columns or as YEAR and DAY-OF-YEAR in separate columns (which we will go over later) is preferable for both clarity and to reduce the chance for errors."

**Comment by morskyjezek** [comment]
Hello @diannamorganti , thank you for these suggestions! It is always helpful to note typos and other errors like this. I think that I can start work to propose a correction to the initial errors you've pointed out. 

To clarify, however, I don't notice any use of "odd" in the page for Episode 3, dealing with dates. It may have been already corrected. If not, please do let us know with a reference to the line where that occurs.

**Comment by morskyjezek** [comment]
Thanks again for this suggestion! We're always interested in feedback and ways to improve Library Carpentry lessons. We've addressed the suggestion in the referenced issue.
